tests after installation of spectrochempy_data

**Describe you problem**

I have installed spectrochempy_data but testing does not work. On  WIN , `preferences.datadir`  returns `OSError: symbolic link privilege not held`

 for instance:

```
C:\Miniconda3\envs\scpy-dev\python.exe "C:\Program Files\JetBrains\PyCharm 2020.2.3\plugins\python\helpers\pycharm\_jb_pytest_runner.py" --path C:/Users/traverta/Dropbox/Programmes/spectrochempy/tests/test_plotters
Testing started at 16:58 ...
Launching pytest with arguments C:/Users/traverta/Dropbox/Programmes/spectrochempy/tests/test_plotters in C:\Users\traverta\Dropbox\Programmes\spectrochempy\tests\test_plotters

SpectroChemPy's API - v.0.2.11.dev2+g6ceb47f0
Loading SpectroChemPy API: 100%|██████████| 1211/1211 [00:02<00:00, 548.16it/s]
© Copyright 2014-2021 - A.Travert & C.Fernandez @ LCS
ImportError while loading conftest 'C:\Users\traverta\Dropbox\Programmes\spectrochempy\tests\conftest.py'.
..\conftest.py:18: in <module>
    import spectrochempy
..\..\spectrochempy\__init__.py:74: in <module>
    from spectrochempy import api
..\..\spectrochempy\api.py:126: in <module>
    DATADIR = pathclean(preferences.datadir)
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:575: in __get__
    return self.get(obj, cls)
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:538: in get
    default = obj.trait_defaults(self.name)
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:1578: in trait_defaults
    return self._get_trait_default_generator(names[0])(self)
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:975: in __call__
    return self.func(*args, **kwargs)
..\..\spectrochempy\application.py:478: in _get_default_datadir
    return self.parent.datadir.path
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:575: in __get__
    return self.get(obj, cls)
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:538: in get
    default = obj.trait_defaults(self.name)
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:1578: in trait_defaults
    return self._get_trait_default_generator(names[0])(self)
C:\Miniconda3\envs\scpy-dev\lib\site-packages\traitlets\traitlets.py:975: in __call__
    return self.func(*args, **kwargs)
..\..\spectrochempy\application.py:418: in _get_path_default
    path.symlink_to(testdata, target_is_directory=True)
C:\Miniconda3\envs\scpy-dev\lib\pathlib.py:1352: in symlink_to
    self._accessor.symlink(target, self, target_is_directory)
E   OSError: symbolic link privilege not held

Process finished with exit code 4
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -82,9 +82,6 @@ install:
   - conda activate scpy$CONDAPY;
   - git fetch --tags;
   - python setup.py install;
-  # check for data
-  - echo "testing data existence in $CONDA_PREFIX/share/spectrochempy_data/testdata"
-  - ls $CONDA_PREFIX/share/spectrochempy_data/testdata/irdata
 
 # command to install code-climate dependencies
 before_script:
@@ -98,6 +95,9 @@ before_script:
 script:
   - if [[ $TEST = true ]]; then
       echo "RUNNING TESTS";
+      # check for data
+      echo "testing data existence in $CONDA_PREFIX/share/spectrochempy_data/testdata";
+      ls $CONDA_PREFIX/share/spectrochempy_data/testdata/irdata;
       flake8;
       coverage run -m pytest -v
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -95,7 +95,6 @@ before_script:
 script:
   - if [[ $TEST = true ]]; then
       echo "RUNNING TESTS";
-      # check for data
       echo "testing data existence in $CONDA_PREFIX/share/spectrochempy_data/testdata";
       ls $CONDA_PREFIX/share/spectrochempy_data/testdata/irdata;
       flake8;

--- a/spectrochempy/__init__.py
+++ b/spectrochempy/__init__.py
@@ -54,22 +54,6 @@ It is a cross platform software, running on Linux, Windows or OS X.
 
 __all__ = ['api']
 
-# if '/bin/scpy_update' not in sys.argv[0]:  # avoid indefinite loop of calls this process
-#
-#     p = Path('~/.spectrochempy/tmp')
-#     p = p.expanduser()
-#     p.mkdir(parents=True, exist_ok=True)
-#     q = p / 'updated'
-#     q.touch(exist_ok=True)
-#     text = q.read_text()
-#     if not text:
-#         text = str(time() - 90000)
-#     if time() - float(text) > 90000:  # check every 24 hours at the maximum
-#         import subprocess
-#
-#         subprocess.Popen(["scpy_update"])
-#         q.write_text(str(time()))
-
 # import the main api
 from spectrochempy import api
 from spectrochempy.api import *  # noqa: F401

--- a/spectrochempy/application.py
+++ b/spectrochempy/application.py
@@ -370,7 +370,9 @@ class DataDir(HasTraits):
         # try to use the conda installed testdata (spectrochempy_data package)
         try:
             conda_env = environ['CONDA_PREFIX']
-            path = Path(conda_env) / 'share' / 'spectrochempy_data'
+            path = Path(conda_env) / 'share' / 'spectrochempy_data' / 'testdata'
+            if not path.exists():
+                path = Path(conda_env) / 'share' / 'spectrochempy_data'   # depending on the version of spectrochempy_data
         except KeyError:
             pass
 

--- a/spectrochempy/application.py
+++ b/spectrochempy/application.py
@@ -359,6 +359,24 @@ class DataDir(HasTraits):
 
     path = Instance(Path)
 
+    @default('path')
+    def _get_path_default(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        # create a directory testdata in .spectrochempy to avoid an error if the following do not work
+        path = _find_or_create_spectrochempy_dir() / 'testdata'
+        path.mkdir(exist_ok=True)
+
+        # try to use the conda installed testdata (spectrochempy_data package)
+        try:
+            conda_env = environ['CONDA_PREFIX']
+            path = Path(conda_env) / 'share' / 'spectrochempy_data'
+        except KeyError:
+            pass
+
+        return path
+
     def listing(self):
         """
         Create a str representing a listing of the testdata folder.
@@ -395,55 +413,6 @@ class DataDir(HasTraits):
     def _repr_html_(self):
         # _repr_html is needed to output in notebooks
         return self.listing().replace('\n', '<br/>').replace(" ", "&nbsp;")
-
-    @default('path')
-    def _get_path_default(self):
-        # the spectra path in package data
-        # OLD: return Path(get_pkg_path('testdata', 'scp_data'))
-        # now installed with spectrochempy_data
-        # we need to fing the share directory
-        path = _find_or_create_spectrochempy_dir() / 'testdata'
-
-        must_copy = False
-        if not path.exists() or (not path.is_symlink() and not list(path.iterdir())):
-            # try to use the conda installed tesdata (spectrochempy_data package)
-            try:
-                conda_env = environ['CONDA_PREFIX']
-                testdata = Path(conda_env) / 'share' / 'spectrochempy_data'
-                if testdata.exists():
-                    # create a symbolic link to this tesdata directory
-                    # However this work locally or on Colab, BUT not on Travis
-                    if not environ.get('TRAVIS_BRANCH', None):
-                        if path.exists():
-                            path.rmdir()
-                        path.symlink_to(testdata, target_is_directory=True)
-                    else:
-                        must_copy=True
-            except KeyError:
-                pass
-
-            except OSError:
-                must_copy = True
-
-            if must_copy:
-                # we need to copy file so it will work
-                from spectrochempy.utils import copytree
-                if path.exists():
-                    path.rmdir()
-                copytree(testdata, path.parent)
-
-
-        if path.exists() and path.is_file():
-            msg = 'Intended Data directory `{0}` is ' \
-                  'actually a file.'
-            raise IOError(msg.format(path))
-
-        # OK but what if like in colab we found nothing
-        if not path.exists():
-            # create a directory to avoid an error
-            path.mkdir()
-
-        return path
 
 
 # ======================================================================================================================

--- a/spectrochempy/application.py
+++ b/spectrochempy/application.py
@@ -21,7 +21,6 @@ import datetime
 import warnings
 import pprint
 import json
-import time
 from os import environ
 from pathlib import Path
 import threading

